### ISSUE TITLE
after instalation err

### ISSUE THREAD

**Issue by naeemq71** [formtools/core#622]
The "index.tpl" template could not be located at the following locations: D: mpp\htdocsormtools/themes/default/index.tpl and D: mpp\htdocsormtools/themes/default/index.tpl.

**Comment by benkeen** [comment]
Hi @naeemq71, hmm... are you on a Windows server? The missing characters in that folder paths look very suspicious. I'll put this down as a tentative bug & investigate today.

**Comment by benkeen** [comment]
Confirmed. On Windows environments the generated config.php file is incorrect. The $g_root_dir folder needs to have the "\" characters escaped, so this:

```php
$g_root_dir = "c:\path\toformtools";
```

Needs to be:

```php
$g_root_dir = "c:\\path\\toformtools";
```

It'll cause the last step of the installation script to fail. Apologies, I'll get a new version out today.

**Comment by benkeen** [comment]
Fixed in 3.0.19.
